vspace/src/main/webapp/resources/extra/Module_1_slideshow_1.css

```diff
@@ -1,217 +1,229 @@
+}
+
+.Back_to_first_Class {
+    position: absolute;
+    width: 44px;
+    height: 44px;
+    left: 20px;
+    top: 198px;
+    overflow: visible; 
+    -web-animation: fadein 0.3s ease-out; 
+    -web-action-type: page; 
+    -web-action-target: Module_1___1.html;
+    cursor: pointer;
+}
+
+.Ellipse_5 {
+    filter: drop-shadow(3px 3px 20px rgba(0, 0, 0, 0.251));
+    position: absolute;
+    overflow: visible;
+    width: 45px;
+    height: 45px;
+    left: 0px;
+    top: 0px;
+}
+
+.Icon_material_first_page {
+    overflow: visible;
+    position: absolute;
+    width: 18px;
+    height: 17.405px;
+    left: 13px;
+    top: 13px;
+    transform: matrix(1, 0, 0, 1, 0, 0);
+}
+
+._DigInG_Class {
+    position: absolute;
+    left: 1206px;
+    top: 693px;
+    overflow: visible;
+    width: 47px;
+    white-space: nowrap;
+    text-align: left;
+    font-family: Helvetica Neue;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 12px;
+    color: rgba(0, 0, 0, 1);
+}
+
+a, a:hover, a:focus {
+    color: inherit;
+    text-decoration: none;
+    transition: all 0.3s;
+}
+
+#dismiss {
+    width: 35px;
+    height: 35px;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+}
+
+#wrapper_module {
+    width: 100%;
+    align-items: center;
+    overflow: hidden;
+}
+
+#slide_sidebar_module {
+    min-width: 250px;
+    max-width: 250px;
+    transition: all 0.3s;
+}
+
+#slide_sidebar_module.active {
+    margin-left: -250px;
+}
+
+#slide_sidebar_module .slide_sidebar-header {
+    padding: 10px;
+}
+
+.slide_sidebar-header h3 a:hover {
+    color: #7386D5;
+    background: #fff;
+}
+
+#slide_sidebar_module ul li {
+    padding-left: 35px;
+}
+
+#slide_sidebar_module ul li a:hover {
+    color: #7386D5;
+    background: #fff;
+}
+
+#slide_content_module {
+    width: 100%;
+    padding: 20px;
+    min-height: 100vh;
+    transition: all 0.3s;
+}
+
+.imgDiv {
+    position: relative;
+    max-width: 57%;
+    top: 0px;
+    overflow: visible;
+    margin: 1%;
+    display: block;
+}
+
+.textDiv {
+    position: relative;
+    width: 57%;
+    top: 0px;
+    overflow: visible;
+    text-align: justify;
+    margin: auto;
+}
+
+.biblioDiv {
+    position: relative;
+    width: 57%;
+    top: 0px;
+    overflow: visible;
+    text-align: justify;
+    margin: auto;
+}
+
+.slideNumberClass {
+    position: absolute;
+    width: 84%;
+    padding-top: 1%;
+    display: flex;
+    justify-content: flex-end; } .Component_31___11_Class {
+    position: absolute;
+    width: 97%;
+    display: flex;
+    justify-content: flex-end;
+}
+
+.Mask_Group_9_Class {
+    position: realtive;
+    width: 34px;
+    height: 34px;
+    left: 0px;
+    top: 0px;
+    overflow: visible;
+    cursor: pointer;
+}
+
+.language-picker {
+    display: inline-block;
+    position: relative;
+}
+
+@media only screen and (max-width: 800px) {
+    /* For mobile phones: */
+    [class*="Group_7_Class"] {
+        width: 98%;
+    }
+}

```

Comment:
it looks like this file has the wrong file endings; please fix your ide settings and revert.